src/components/Hero.tsx

```diff
@@ -104,9 +105,13 @@ const Hero = () => {
 
                     <div className="flex justify-between mt-[8px]">
                       <div className="flex gap-x-[5px]">
-                        <div className="w-[20px] h-[20px] rounded-full">
-                          {/* todos: add contributors avator here */}
-                        </div>
+                        {contributions.slice(0, 3).map(contribution => (
```

Comment:
Please use destructuring `.map(({contributor, last_merged_at}))`

Reply:
> Thanks for opening this @ddsuhaimi. Could you assist in making the avatars the same size as how they are in the other cards and bring in the hover effects. You should be able to copy the classes in [this div](https://github.com/open-sauced/hot/blob/main/src/components/HotRepositories.tsx#L128).

so instead of addressing the above in this PR, we can address in this issue https://github.com/open-sauced/hot/issues/287